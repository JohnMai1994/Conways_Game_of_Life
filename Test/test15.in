new 30
init
10 10 10 11 10 12 10 13 10 14
-1 -1
print
step
print
step
step
step
print
steps 5
print

