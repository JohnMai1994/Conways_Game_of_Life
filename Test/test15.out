______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
__________X___________________
__________X___________________
__________X___________________
__________X___________________
__________X___________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
_________XXX__________________
_________XXX__________________
_________XXX__________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
_________XXX__________________
________X___X_________________
________X___X_________________
________X___X_________________
_________XXX__________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
__________X___________________
__________X___________________
__________X___________________
______________________________
______XXX___XXX_______________
______________________________
__________X___________________
__________X___________________
__________X___________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
______________________________
